### PR TITLE
[EventGrid/ACR] Add schema of ACR events published to Event Grid

### PR DESCRIPTION
**Pull request by djyou**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        "title": "Schema of Azure Container Registry events published to Azure Event Grid",
+        "description": "Describes the schema of the Azure Container Registry events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+    },
+    "paths": {},
+    "definitions": {
+        "ContainerRegistryImagePushedEventData": {
+            "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ContainerRegistry.ImagePushed event.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/EventContent"
```
Comment (kalyanaj) [inline specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: For all the types referenced here (EventContent, Actor, Target, etc.), it looks to me that they should be qualified with the prefix "ContainerRegistry" as all these types will get generated under the same namespace Microsoft.Azure.EventGrid.Models. @lmazuel, what do you think, is there a better approach?

Comment (djyou) [reply specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: `EventContent` can be prefixed with `ContainerRegistry` for sure. We had a similar issue with the naming here in our own models that types such as `Target`, `Request`, and `Source` are from standard `docker` notification schema but they may conflict with other models in the same namespace.

Are you generating data-plane models into SDKs?

**Review by kalyanaj — COMMENTED** [review]
Looks good to me.

**Review by lmazuel — APPROVED** [review]
LGTM

**Pull request merged**